app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -264,10 +265,12 @@ class ChatActivity : AppCompatActivity(), IChatView {
 
     //Replies user with Speech
     override fun voiceReply(reply: String, language: String) {
-        val audioFocus = getSystemService(Context.AUDIO_SERVICE) as AudioManager
+        val mAudioFocus = getSystemService(Context.AUDIO_SERVICE)
+        var audioFocus: AudioManager? = null
+        if (mAudioFocus is AudioManager) audioFocus = mAudioFocus
```

Comment:
Don't use hungarian notation. If it is not AudioManager, just return

Reply:
Updated